warehouse/warehouse/scripts/move_to_publication.py

```diff
@@ -156,8 +156,7 @@ def main():
                 outstream.write(message)
                 con_message("info", message)
         else:
-            con_message("error", f"cannot obtain message_file (from message_path) from environment for message {message}")
-            sys.exit(1)
+            con_message("info", f"cannot obtain message_file (from message_path) from environment for message {message}")
```

Comment:
Does `con_message()` support `"warning"`?

Reply:
Thanks Tom.  I'd been toggling "info", and "debug" for so long I'd forgotten "warning".